.env.example

```diff
@@ -1,5 +1,6 @@
 OPENAI_KEY=
 ELASTIC_URL=
+ELASTIC_CLOUD_ID=
```

Comment:
I forgot, is there a way of commenting in `.env` files? Im worried people will think they need to supply both id and username/password.